content/_data/authors/iamrajiv.adoc

```diff
@@ -3,5 +3,5 @@ name: "Rajiv Ranjan Singh"
 blog: https://iamrajiv.github.io
 github: iamrajiv
 linkedin: iamrajivranjansingh
-twitter: therajiv
+x: therajiv
```

Comment:
```suggestion
twitter: therajiv
```

Reply:
Hi @iamrajiv,
We are still using "twitter" in the codebase for the social media now known as "X".